src/analysis/undocumented.d

```diff
@@ -133,7 +133,10 @@ class UndocumentedDeclarationCheck : BaseAnalyzer
 	{
 	}
 
+	mixin V!AnonymousEnumMember;
 	mixin V!ClassDeclaration;
+	mixin V!EnumDeclaration;
+	mixin V!EnumMember;
```

Comment:
I like the addition of AnonymousEnumMember and EnumDeclaration, but the check for enum members may not add as much value.